turbo/consumers.py

```diff
@@ -31,62 +28,37 @@ def notify(self, event, *args, **kwargs):
         else:
             dom_target = f"{model._meta.verbose_name.lower()}_{pk}"
 
-        for request_id in self.requests[event["channel_name"]]:
```

Comment:
This is the only real change in this block of code. The rest is just unindented.